lib/rspec/bash/wrapper/stub_function.rb

```diff
@@ -0,0 +1,36 @@
+module Rspec
+  module Bash
+    class StubFunction
+      def initialize(port, stub_script_class)
```

Comment:
Not a huge fan of passing in the parameterized class. I'd like to get that reworked at some point.